### DISCUSSION TITLE
Adding fixed deduction on the income

### DISCUSSION THREAD

**Discussion by njmulsqb** [frappe/books#729] [Q&A]
Hi,

I am new to Frappe and am using it for my software company; I am really liking this software so thanks for building it.

On my income, I usually give 10% to charity, is there any way I can add this to Frappe, so that it shows me the amount I need to give to charity from my income and also shows the profit after deducting the charity?

**Comment by 18alantom** [comment]
You could use a Journal Entry to record arbitrary transactions such as donating to charity. Depending on how you want to record it, you might add your cash or bank account on the debit side. And the charity account on the credit side.


You'll have to calculate the amount (10%) manually.

**Comment by njmulsqb** [comment]
Alright, one more question, in my case the final revenue gets divided in  three persons, in some case it is equal division, while in other it will be 50%,25%,25%. Is there any way in Frappe I can get this information auto-calculated? Maybe using custom form or using Chart of Accounts?

**Reply by 18alantom** [reply]
In Frappe Books there's no user side support for adding custom forms, you can customize existing forms but you can't add logic. If you're a developer (or know one) a custom form that allows for revenue division can be created.

Else, a non development work around is to create a Journal Entry for the split and create duplicate and submit them every time you need to record such split. You can use duplicate like a templating feature this way.
